blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/FavoritesStatePreloader.java

```diff
@@ -54,9 +60,10 @@
     @Override
     protected FetchData getFetchData(@Nonnull BlueUrlTokenizer blueUrl) {
         User jenkinsUser = User.current();
+        BlueOrganization organization = Iterables.getFirst(OrganizationFactory.getInstance().list(), null);
```

Comment:
I would definitely just add a method to `OrganizationFactory` called something like `getDefault()` that returns whatever the "default" organization is, and ideally is `@Nonnull`